packages/ndla-ui/src/Notion/ConceptNotion.tsx

```diff
@@ -4,34 +4,78 @@
  * This source code is licensed under the GPLv3 license found in the
  * LICENSE file in the root directory of this source tree. *
  */
+import styled from '@emotion/styled';
 
-import React, { useEffect } from 'react';
+import React, { useEffect, Fragment } from 'react';
 //@ts-ignore
 import { initArticleScripts } from '@ndla/article-scripts';
+
+import NotionDialog, {
+  NotionDialogContent,
+  NotionDialogText,
+  NotionDialogImage,
+  NotionDialogLicenses,
+} from '@ndla/notion';
+import { breakpoints, mq, spacing } from '@ndla/core';
 import { Notion } from '.';
 import { NotionImage } from './NotionImage';
 import NotionVisualElement, { NotionVisualElementType } from './NotionVisualElement';
 import FigureNotion from './FigureNotion';
 import { Copyright } from '../types';
 
+const ContentWrapper = styled.div<{ adjustSizeToFitWiderPage?: boolean }>`
+  position: relative;
+  right: auto;
+  left: ${(props) => (props.adjustSizeToFitWiderPage ? '0%' : '-16.6666666667%')};
+  width: ${(props) => (props.adjustSizeToFitWiderPage ? '100%' : '133.3333333333%')};
```

Comment:
I hvilke tilfeller trenger man 133% width?

Reply:
Trengs for å bli bredere enn artikkelens container. 
<img width="913" alt="image" src="https://user-images.githubusercontent.com/25527932/167089632-e49d7b45-30a4-4b2d-aec9-245cb891e56a.png">